assets/prototype/domain/eventEditor/datesList/dateCard/DateCard.js

```diff
@@ -29,11 +33,8 @@ const idStyle = {
 const DateCard = ({ eventId, id }) => {
 	const date = useDateItem({ id });
 	const onFieldUpdate = useUpdateDateMutation({ id });
-
-	const startDate = date.start ? new Date(date.start * 1000) : PLUS_ONE_MONTH;
-	// const endDate = date.end ?
-	// 	new Date( date.end * 1000 ) :
-	// 	PLUS_TWO_MONTHS;
+	const defaultRangeValues = [moment(date.startDate).toDate() || null, moment(date.endDate).toDate() || null];
+	const [range, setRange] = useState(defaultRangeValues);
```

Comment:
I've used moment here in order to try to fed `DateRangePicker` with default values.

Reply:
let's use:
- `PLUS_ONE_MONTH` instead of `null` for the default `startDate`
- `PLUS_TWO_MONTHS` instead of `null` for the default `endDate`

Reply:
Done, thx for suggestion.